manim/cli/render/commands.py

```diff
@@ -136,4 +137,43 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version:
+        manim_info_url = "https://pypi.org/pypi/manim/json"
+        warn_prompt = "Cannot check if latest release of manim is installed"
+        req_info = {}
+        releases = {}
```

Comment:
Since this is only used in one place, you could just remove the `releases` variable and plug it straight into `max()` to get the stable release.